code/style.css

```diff
@@ -1,7 +1,333 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+body {
+  margin: 0;
+  background-color: lightgrey;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+/***header***/
+header {
+  position: relative;
+  margin: 0 auto;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  overflow: hidden;
+  margin-top: 0;
+}
+
+div.header-text h1 {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+  font-size: 90px;
+  opacity: 0.6;
+  color: #fdffb6;
+}
+
+.header-text span {
+  position: absolute;
+  top: 70%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+  font-size: 40px;
+  opacity: 0.6;
+  color: #fdffb6;
+}
+
+.video-container {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  overflow: hidden;
+  z-index: -1;
+}
+
+video {
+  object-fit: fill;
+  width: 100%;
+  height: 100vh;
+  z-index: 0;
+}
+
+picture {
+  display: none;
+}
+
+nav {
+  position: absolute;
+  width: 50%;
+  top: 0;
+  background-color: transparent;
+  margin: 10px;
+  overflow: hidden;
+}
```

Comment:
When making the screen smaller the navbar kind of disappears into the right side, check it out and see what you can do about it 🚀